lib/straight/blockchain_adapters/chain_com_adapter.rb

```diff
@@ -0,0 +1,81 @@
+module Straight
+  module Blockchain
+
+    class ChainComAdapter < Adapter
+
+      def self.mainnet_adapter(api_key_id:)
+        instance = self.instance
+        instance._initialize('https://api.chain.com/v2/bitcoin', api_key_id)
+        instance
+      end
+
+      def self.testnet_adapter(api_key_id:)
+        instance = self.instance
+        instance._initialize('https://api.chain.com/v2/testnet3', api_key_id)
```

Comment:
Yeah, you're right. I haven't seen it. I thought, we have unique initialization for the whole project. I'll fix it.